Dani_test_pr

danisoloo opened the pull request:
<!--

You must title your PR like this:

London | Sepe 25 ITP | Daniel Solomon | Sprint 1 | Wireframe

For example,

London | 25-ITP-May | Carol Owen | Sprint 1 | Alarm Clock

Fill in the template below - remove any sections that don't apply.

Complete the self checklist - replace each empty box in the checklist [ ] with a [x].

Add the label "Needs Review" and you will get review.

Respond to volunteer reviews until the volunteer marks it as "Complete".

-->

## Learners, PR Template

Self checklist

- [X ] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [X ] My changes meet the requirements of the task
- [X ] I have tested my changes
- [X ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

cjyuan reviewed (commented):
1. It seems you have modified the wrong `index.html`.
You were supposed to modify the `index.html` inside the Wireframe folder.
Can you also restore the original `index.html` in the top-level folder (to keep your branch clean)?

2. Your placeholder images do not seem to work. You can use the `placeholder.svg` that's already in the Wireframe folder in this exercise.

3. In the PR description, can you restore the "Changelist" section and add a brief description about the changes you made in your PR?

danisoloo commented:
ok what about my form control?


On Sat, Sep 27, 2025 at 10:42 AM CJ Yuan ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
>    1.
>
>    It seems you have modified the wrong index.html.
>    You were supposed to modify the index.html inside the Wireframe folder.
>    2.
>
>    Your placeholder images do not seem to work. You can use the
>    placeholder.svg that's already in the Wireframe folder in this
>    exercise.
>    3.
>
>    In the PR description, can you restore the "Changelist" section and
>    add a brief description about the changes you made in your PR?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/827#pullrequestreview-3274768530>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BTQPWO7432CCGRZS4F3SIQT3UZLZJAVCNFSM6AAAAACG73FSGWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTENZUG43DQNJTGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

cjyuan reviewed (commented):
Page looks good now.

Can you fix the "Changelist" heading in your PR description?

danisoloo commented:
Yes, but I’m using my Phone and I didn't get the label, I’m figuring it out.

On Sunday, September 28, 2025, CJ Yuan ***@***.***> wrote:

> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#827)
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/827#issuecomment-3342217615>
>
> You can try these in future PRs.
>
>    1.
>
>    Whenever you have addressed the reviewer's feedback (even if the
>    request does not involve changing any code), you should change the label to
>    "Needs review" to indicate the changes are ready to be reviewed.
>    2.
>
>    The "Changelist" in the PR description is for describing what this PR
>    is about (instead of what you have changed in respond to the reviewer's
>    comment). To learn more about pull request, you can try asking ChatGPT what
>    a PR is, and what do developers usually write in the Changelist section in
>    the PR description?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/827#issuecomment-3342217615>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BTQPWO2TZ4YA763ZISTG4VL3U5EYFAVCNFSM6AAAAACG73FSGWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGNBSGIYTONRRGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

cjyuan commented:
Have you filled in the form in this **backlog** (in Sprint 1)?

If so, have you clicked the link in an email join the CodeYourFuture group on GitHub (so that you can use labels on your PRs)?
The invitation email is sent by GitHub. You can search "Invite" in your email account to locate the email.

cjyuan commented:
Changes look good. Well done!

danisoloo commented:
I have created new issue and submitted it in step 1 on the course
platform.

Thanks a lot.

> Message ID: ***@***.***
> com>
>
